client/src/modules/list/model/reducer.js

```diff
@@ -73,6 +73,19 @@ const membersReducer = (state = {}, action) => {
         [userId]: { ...state[userId], isOwner: false, isMember: false }
       };
     }
+    case ListActionTypes.MEMBER_UPDATE_SUCCESS: {
+      const {
+        payload: { userId, listId, isCurrentUserUpdated, ...data }
+      } = action;
+      const previousMember = state[userId];
+      const dataToUpdate = _pickBy(data, el => el !== undefined);
```

Comment:
The pattern is replicated many times (dataToUpdate few lines below, for example), consider using object filter helper like this:
```javascript
const filter = f => xs => 
  Object.fromEntries(
    Object.entries(xs)
      .filter(([key, value]) => f(value, key, xs))
  );
``` 

Note: Object.fromEntries might need a polyfill, however, reduce-based implementation is not that hard.

Then you can define and reuse common helpers:

```javascript
const defined = x => x !== undefined;
const filterDefined  = filter(defined);

...

const dataToUpdate  = filterDefined(data);
```